policyengine_us/variables/gov/ed/pell_grant/efc/pell_grant_simplified.py

```diff
@@ -0,0 +1,18 @@
+from policyengine_us.model_api import *
+
+
+class pell_grant_simplified(Variable):
```

Comment:
to make it sound more bool-y
```suggestion
class pell_grant_simplified_formula_applies(Variable):
```